src/test/java/MainTest.java

```diff
@@ -0,0 +1,69 @@
+import exception.NotEnoughWordsInList;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+public class MainTest {
```

Comment:
@joej0e test class name must conform to the convention  https://mate-academy.github.io/style-guides/java/java.html#s5.2.2-class-names